### PR TITLE
first part of translate tutorial add query string to URL and drop dow…

### PR DESCRIPTION
**Pull request by cguruceaga**
…n menu

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/data/Comment.java [hunk 1]
```diff
@@ -5,11 +5,11 @@
 
   private final long id;
   private final String comment;
-  private final long timestamp;
   private final String firstname;
   private final String lastname;
+  private final long timestamp;
 
-  public Comment(long id, String comment, String lastname, String firstname, long timestamp) {
+  public Comment(long id, String comment, String firstname, String lastname, long timestamp) {
```
Comment (matty377) [inline portfolio/src/main/java/com/google/sps/data/Comment.java]: In Java, identifiers are typically camel-case, meaning they start with a lowercase letter but each word after that the first letter is uppercase. So, firstName and lastName in this case.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -30,18 +30,22 @@
...
+import com.google.cloud.translate.Translate;
+import com.google.cloud.translate.TranslateOptions;
+import com.google.cloud.translate.Translation;
 
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/comment")
 public class DataServlet extends HttpServlet {
   private ArrayList<Comment> comments = new ArrayList<>();
+  private ArrayList<String> mycomments = new ArrayList<>();
```
Comment (matty377) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: You can just assign this into a `List` to keep it more generic.

`private List<String> myComments = new ArrayList<>();`

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -51,30 +55,32 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
...
-    long timestamp = System.currentTimeMillis(); 
 
     Entity taskEntity = new Entity("Comments");
-    taskEntity.setProperty("comment", text);
-    taskEntity.setProperty("timestamp", timestamp); 
-    taskEntity.setProperty("first name", first);
-    taskEntity.setProperty("last name", last);
-
+    taskEntity.setProperty("text", text);
+    mycomments.add(text);
```
Comment (matty377) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Are these just getting saved into memory? I see the field on the class but I think these should get looked up each time.

Comment (cguruceaga) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: deleted mycomments list that was just for testing

**Review by matty377 — APPROVED** [review]
LGTM overall but I think you accidentally committed a week 4 file.

**Pull request merged**